### ISSUE TITLE
Search Providers / Provider Options / Minimum Seeders = 1 --> most search results inaccessible

### ISSUE THREAD

**Issue by w4hnsinn** [pymedusa/Medusa#4414]
For the longest time I've been scratching my head why certain (more "rare") releases are simply not shown in medusa, despite the fact that these releases ARE present on some of my search provider sites.

So, turns out that many search providers report incorrect numbers of seeders, namely "0".
Medusa values this info *very* highly, apparently. Cannot even access such releases through "forced search". They just don't show up no matter what.

Now that I've set all my search providers to "Minimum seeders: 0", I get the results I expected from my regular search providers.

**Comment by medariox** [comment]
We can only work with the information we get from the providers. If you have set up minimum seeders with `1` and the providers report releases with fewer seeders, these releases won't be taken into consideration. For providers that report the numbers of seeders correctly, it will cause to snatch releases with `0` seeders, which obviously would never finish the download. There is really not much we can do about it - what you can do is change the minimum seeders to what you prefer, like you now did.

**Comment by w4hnsinn** [comment]
I simply think these results should not be scrubbed from forced/manual searches. Give users the option. That to me is the point of forced&manual searching.

**Comment by sharkykh** [comment]
@medariox 
Could be related to https://github.com/pymedusa/Medusa/blob/ebcbdd5e0b83ab77d7df2f9190cf817038a522e7/medusa/providers/torrent/html/limetorrents.py#L156-L161
We should be removing all of those checks from the providers, correct?
As it's already done when picking a result to download:
https://github.com/pymedusa/Medusa/blob/ebcbdd5e0b83ab77d7df2f9190cf817038a522e7/medusa/search/core.py#L298-L311

**Comment by medariox** [comment]
@sharkykh 
Yes, it should be indeed possible to remove that check in every provider. We should try it, especially now that the code is a bit more readable.
